GVRf/Sample/scene-objects/src/org/gearvrf/sample/sceneobjects/SampleViewManager.java

```diff
@@ -131,12 +172,15 @@ public void onTap() {
             currentObject = 0;
         }
 
-        objectList
-                .get(currentObject)
-                .getRenderData()
-                .setRenderMask(
-                        GVRRenderData.GVRRenderMaskBit.Left
-                                | GVRRenderData.GVRRenderMaskBit.Right);
+        object = objectList.get(currentObject);
+        if (object instanceof GVRVideoSceneObject) {
+            GVRVideoSceneObject video = (GVRVideoSceneObject) object;
+            video.getMediaPlayer().start();
+        }
+        
+        object.getRenderData().setRenderMask(
+                GVRRenderData.GVRRenderMaskBit.Left
```

Comment:
Agreed.  we should do that in a separate check-in for GVRRenderData.  We should add a 'both' and 'none' or 'off' or something.